src/Raven.Server/Documents/Handlers/MultiGetHandler.cs

```diff
@@ -184,16 +185,17 @@ public async Task PostMultiGet()
                             }
                         }
                         writer.WriteEndObject();
-
                         writer.WriteEndObject();
+                        // have to be at the end of the for
+                        if (TrafficWatchManager.HasRegisteredClients)
+                        {
+                            HttpContext.Items["TrafficWatch"] = (string)HttpContext.Items["TrafficWatch"] + content.ToString() + "\n";
```

Comment:
Use `StringBuilder` here and set it once

Reply:
done